spec/prawn/text_spec.rb

```diff
@@ -66,7 +66,7 @@
       pdf.text ' '
       text = PDF::Inspector::Text.analyze(pdf.render)
       # If anything is rendered to the page, it should be whitespace.
-      text.strings.each { |str| expect(str).to match(/\A\s*\z/) }
+      expect(text.strings).to all(match(/\A\s*\z/))
```

Comment:
I'm not opposing this change but it doesn't seem to be related to misspellings in any way. Maybe at least a separate commit?